client/blocks/login/two-factor-authentication/verification-code-form.scss

```diff
@@ -1,11 +1,24 @@
 .two-factor-authentication__small-print {
 	clear: both;
-	font-size: $font-body-extra-small;
-	padding: 1em 0;
+	padding: 2em 0 0 0;
+	text-wrap: pretty;
 }
 
-.two-factor-authentication__verification-code-form .button {
-	width: 100%;
+.two-factor-authentication__verification-code-form {
+	.button {
+		width: 100%;
+	}
+
+	.form-label {
+		text-transform: uppercase;
```

Comment:
@ciampo @tyxla I'm curious if there's a better way to do this than replicate the visual appearance of the core label style. Do we get something for free if we import the BaseControl component? I forget the details.

Reply:
Unfortunately, if we want to do this consistently, it would be part of a larger refactor. Right now, this uses `@automattic/components`'s `FormLabel`, which is a completely separate component. 

But to answer your question partially, yes, we do get some of that for free if we use `BaseControl`:

https://github.com/WordPress/gutenberg/blob/3bde697156b5dde4c2b9c12c1e238d0e5dce5ead/packages/components/src/utils/base-label.ts#L8-L13